app/views/new_threads/_search.html.erb

```diff
@@ -1,4 +1,4 @@
- <%= form_tag({:controller => '/new_threads', :action => 'search'}, {:id => 'search'}) do %>
+ <%= form_tag({:controller => '/new_threads', :action => 'search'}, {:id => 'search', :class => 'navbar-form navbar-right'}) do %>
      <%= text_field_tag :query, params[:query], :id => 'search_field', :placeholder => "Search threads..." %>
      <%= submit_tag "Search", :name => nil, :id => 'search_button', :style => "display: none;" %> 
```

Comment:
@ghoshbishakh you still have to remove the display:none property and put it in stylesheet